### PR TITLE
Update OpenTelemetry dependency to 0.14.1

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Comment by jpkrohling** [comment]
@kevinearls would you mind reviewing this one?

File: src/main/java/io/observatorium/opentelemetry/generate/account/TracerInitializer.java [hunk 1]
```diff
@@ -1,44 +1,47 @@
...
-                .build();
+        OtlpGrpcSpanExporter spanExporter = OtlpGrpcSpanExporter.builder().setTimeout(2, TimeUnit.SECONDS).build();
+        BatchSpanProcessor spanProcessor = BatchSpanProcessor.builder(spanExporter)
+                .setScheduleDelay(100, TimeUnit.MILLISECONDS).build();
 
-        OtlpGrpcSpanExporter spanExporter = OtlpGrpcSpanExporter.getDefault();
-        BatchSpanProcessor spanProcessor = BatchSpanProcessor.newBuilder(spanExporter).setScheduleDelayMillis(100)
-                .build();
-        sdkProvider.addSpanProcessor(spanProcessor);
+        sdkProvider = OpenTelemetrySdk.builder()
```
Comment (kevinearls) [inline src/main/java/io/observatorium/opentelemetry/generate/account/TracerInitializer.java:33]: One little nit - shouldn't these 4 lines be aligned?  (from .setTracerProvider down to .buildAndRegisterGlobal())  

I haven't written much Java lately so I don't know what current conventions are.

**Review by kevinearls — APPROVED** [review]

File: src/main/java/io/observatorium/opentelemetry/generate/account/TracerInitializer.java (hunk 1 above)
Comment (jpkrohling) [reply src/main/java/io/observatorium/opentelemetry/generate/account/TracerInitializer.java:33]: I haven't done much Java myself either, which is why I let the editor to format it for me :-)

Comment (kevinearls) [reply src/main/java/io/observatorium/opentelemetry/generate/account/TracerInitializer.java:33]: I suppose there's some reason the editor did it that way but I can't imagine what it was.  In any case, it's not a big deal to me.

**Pull request merged**